src/Resources/public/styles/_cookiebar.scss

```
@@ -73,10 +62,7 @@
   }
 
   &.cc-middle{
-
-    .cc-inner {
-      vertical-align: middle;
-    }
+	align-items: center;
```

```suggestion
	align-items: center;
	justify-content: center;
```

Could just add the justify-content: center here